pkg/aws/cloudwatch/histograms/README.md

```diff
@@ -0,0 +1,14 @@
+# Histograms
+
+This package holds common CloudWatch histogram functionality for AWS owned OpenTelemetry components.
+
+## Visualize histogram mappings
+
+1. Remove `t.Skip(...)` from `TestWriteInputHistograms` and run the test to generate json files for the input histograms.
+1. Remove `t.Skip(...)` from `TestWriteConvertedHistograms` and run the test to generate json files for the converted histograms.
```

Comment:
nit: Could hide them behind a go:build flag, so you don't need to modify the code to be able to run them.